<?xml version="1.0"?>
<launch>
	<arg name="camera_topic_in" default="/realsense_img" />
	<arg name="camera_topic_out" default="/realsense_obj" />
<!-- for detect postprocese  used in trafficlight-->
	<arg name="obj_type" default="traffic_light" />


	<node pkg="realsense" type="realsense_node"	name="camera" output="screen">	
		<param name="ImageObj_topic" value="$(arg camera_topic_out)" />
	</node>
	<node pkg="realsense" type="trafficlight_node"	name="trafficlight" output="screen">	
		<param name="light_topic_in" value="$(arg camera_topic_out)" />
		<param name="obj_type" value="$(arg obj_type)"/>
		<param name="light_type" value="0"/>
	</node>

</launch>